plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -465,7 +465,14 @@ function getServiceDocument($journalId) {
 		);
 		
 		// stop here if we didn't get an OK
-		if ($result['status'] != PLN_PLUGIN_HTTP_STATUS_OK) return $result['status'];
+		if ($result['status'] != PLN_PLUGIN_HTTP_STATUS_OK) {
+			if($result['status'] === FALSE) {
+				error_log(__('plugins.generic.pln.error.network.servicedocument') . ' ' . $result['error']);
```

Comment:
Rather than forcing the message to be appended, better to use a variable in the locale key. That way the manner of attaching the error message can be language-specific (think RTL languages).